src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/HelpMessage.java

```diff
@@ -0,0 +1,28 @@
+package io.jenkins.plugins.report.jtreg.main.comparator;
+
+final public class HelpMessage {
+    public static final String HELP_MESSAGE =
+           //80-characters-long-comment------------------------------------------------------
+            "Test Variant Comparator Usage: java -cp <classpaths> <mainclass> [arguments...]\n" +
+            "\n" +
+            "Where mainclass is:\n" +
+            "    io.jenkins.plugins.report.jtreg.main.comparator.VariantComparator\n" +
+            "\n" +
+            "And arguments include:\n" +
+            "\n" +
+            "    These two arguments are mandatory:\n" +
+            "    --path <path/to/jenkins/jobs>\n" +
+            "                  a system path to a directory with your jenkins jobs\n" +
+            "    --query <querystring>\n" +
+            "                  a query string to filter the jobs (The syntax is described\n" +
```

Comment:
This is not nice.  It duplictes switches occurences here, and in switch.
Usually it is better to have some clase arguemtn, and then you cna ruse it here, and in switch.

But otherwise very very thankyou!